vertx-mod-jersey/src/main/java/com/englishtown/vertx/jersey/impl/DefaultJerseyConfigurator.java

```diff
@@ -263,6 +264,13 @@ protected ResourceConfig getResourceConfig() {
             }
         }
 
+        final JsonObject resourceConfigAdditions = config.getObject(CONFIG_RESOURCE_CONFIG);
+        if ( resourceConfigAdditions != null ) {
```

Comment:
Do you mind updating this to be consistent with the code style in the rest of the project?